Backend/DataAccessLayer/Repositories/PointRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using System;
+using System.Data.Entity;
+using System.Linq;
+
+namespace DataAccessLayer.Repositories
+{
+    public class PointRepository
+    {
+        private Point CheckPointLimits(Point point)
+        {
+            if (point.Longitude > 180 || point.Longitude < -180 ||
+                point.Latitude > 90 || point.Latitude < -90)
+                return null;
+            return point;
+        }
+
+        public bool ExistingPoint(DatabaseContext _db, Point point)
+        {
+            var result = GetPoint(_db, point.Id);
+            if (result != null)
+            {
+                return true;
+            }
+            return false;
+        }
+        public Point CreatePoint(DatabaseContext _db, Point point)
+        {
+            if (CheckPointLimits(point) == null)
+                return null;
+            point.UpdatedAt = DateTime.UtcNow;
+                _db.Points.Add(point);
+            return point;
+        }
+
+        public Point GetPoint(DatabaseContext _db, float longitude, float latitude)

```

Comment:
I don't think this function is useful. There's never a case where we're going to query for a point by it's exact longitude and latitude. Even if we did, multiple points can have the same latitude and longitude - you're only grabbing a single point here. Even if we _do_ want this function, this implementation is flawed.

Reply:
So we are only looking up points by Id?

Reply:
Yep

Reply:
Actually, there will be two other ways we'll be looking up points:
1. Radius search based on a latitude/longitude
2. Box search based on two latitude/longitudes

Reply:
Shall I get started on these two additional lookup methods?